lib/config/configuration.go

```diff
@@ -942,13 +942,13 @@ func applyAuthConfig(fc *FileConfig, cfg *servicecfg.Config) error {
 	}
 
 	// read in and set the license file path (not used in open-source version)
-	licenseFile := fc.Auth.LicenseFile
-	if licenseFile != "" {
-		if filepath.IsAbs(licenseFile) {
-			cfg.Auth.LicenseFile = licenseFile
-		} else {
-			cfg.Auth.LicenseFile = filepath.Join(cfg.DataDir, licenseFile)
-		}
+	switch licenseFile := fc.Auth.LicenseFile; {
+	case licenseFile == "":
+		cfg.Auth.LicenseFile = filepath.Join(cfg.DataDir, defaults.LicenseFile)
```

Comment:
In the PR description, it says:
> when empty, /var/lib/teleport/license.pem is used

But this is looking in `cfg.DataDir` not `/var/lib/teleport`. Is the description correct? If `DataDir` is set, and no license file is specified, it is reasonable to look in `DataDir` for it, but that's not what the PR description says.

Or should this be `defaults.DataDir` and not `cfg.DataDir` ?

Reply:
The PR description is slightly wrong, the PR code matches what the docs say:
> The `license_file` path can be either absolute or relative to the configured `data_dir`. If the license file path is not set, Teleport will look for the `license.pem` file in the configured `data_dir`, which is `/var/lib/teleport` by default.